airflow/models/taskmixin.py

```diff
@@ -309,4 +311,4 @@ def has_mapped_dependants(self) -> bool:
         only contains operators, not task groups. In the future, we should
         provide a way to record an DAG node's all downstream nodes instead.
         """
-        return any(self.mapped_dependants())
+        return next(self.mapped_dependants(), None) is not None
```

Comment:
Why this change? `any()` seems clearer to me than calling `next()`

Reply:
It’s _marginally_ faster. I’ll revert.